test/preload-hover.test.js

```diff
@@ -8,3 +8,10 @@ beforeEach(() => {
 it('should construct', () => {
   new PreloadHover();
 });
+
+it('Creates link based on value passed from parameter', () => {
+  const defaultScope = [document.querySelector('#one')]
```

Comment:
`document.querySelector('#one')` in this instance shouldn't match anything, since the `document.body` should be empty at this point. You may want to switch this to:

```javascript
const defaultScope = document.createElement('div');
```